packages/app/src/components/DownloadMenu/DownloadMenu.js

```diff
@@ -118,222 +89,191 @@ export class DownloadMenu extends Component {
             columns,
         })
 
+        this.toggleMenu()
+
         window.open(url, format === 'html' ? '_blank' : '_top')
     }
 
     graphicsMenuSection = () => (
-        <div>
-            <ListSubheader component="div">{i18n.t('Graphics')}</ListSubheader>
-            <MenuItem onClick={this.downloadImage('png')}>
-                <ListItemIcon>
-                    <ImageIcon />
-                </ListItemIcon>
-                <ListItemText>{i18n.t('Image (.png)')}</ListItemText>
-            </MenuItem>
-            <MenuItem onClick={this.downloadImage('pdf')}>
-                <ListItemIcon>
-                    <PictureAsPdfIcon />
-                </ListItemIcon>
-                <ListItemText>{i18n.t('PDF (.pdf)')}</ListItemText>
-            </MenuItem>
-        </div>
+        <Fragment>
```

Comment:
I think you can just use <> now, instead of needing to write <Fragment>.

Reply:
Fixed.